easybuild/easyconfigs/p/pyWannier90/pyWannier90-2021-12-07-gomkl-2021a.eb

```
@@ -0,0 +1,76 @@
+easyblock = 'MakeCp'
+
+name = 'pyWannier90'
+local_commit = '8354e91'
+version = '2021-12-07'
+
+homepage = 'https://github.com/hungpham2017/pyWannier90'
+description = "A Wannier90 Python interface for VASP and PySCF"
+
+toolchain = {'name': 'gomkl', 'version': '2021a'}
+
+local_wannier90_version = '3.1.0'
+
+sources = [
+    {
+        'source_urls': ['https://github.com/hungpham2017/pyWannier90/archive/'],
+        'download_filename': '%s.tar.gz' % local_commit,
+        'filename': SOURCE_TAR_GZ
+    },
+    {
+        'source_urls': ['https://github.com/wannier-developers/wannier90/archive/'],
+        'download_filename': 'v%s.tar.gz' % local_wannier90_version,
+        'filename': 'Wannier90-%s.tar.gz' % local_wannier90_version,
+    },
+]
+checksums = [
+    {'pyWannier90-2021-12-07.tar.gz': '27233891fe39e0f45914dfa756b28319ebd4c3b5ec2124f4c9d1d3816fc0887e'},
+    {'Wannier90-3.1.0.tar.gz': '40651a9832eb93dec20a8360dd535262c261c34e13c41b6755fa6915c936b254'},
+]
+
+builddependencies = [
+    ('pybind11', '2.6.2'),
+]
+
+local_pymaj = '3'
+local_pymin = '9'
+local_pymajmin = local_pymaj + local_pymin
+
+dependencies = [
+    ('Python', '%s.%s.5' % (local_pymaj, local_pymin)),
```

as in #16447, the `local_py{maj,min,majmin}` variables seem redundant
```suggestion
dependencies = [
    ('Python', '3.9.5'),
```

---

fixed in 2f5588a, I think these are leftovers from an earlier draft where I actually did need to version digits separately...